From 66ff2d8a4af06bb3e990d0838e44cca5c5944ba1 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyfox@gentoo.org>
Date: Sat, 25 Jan 2020 17:59:53 +0000
Subject: [PATCH] Common/Hashmaps.h: add missing <stdint> include

On gcc-10 the build fails as:

```
[ 51%] Building CXX object CMakeFiles/Core.dir/GPU/Null/NullGpu.cpp.o
In file included from ../GPU/Common/DrawEngineCommon.h:24,
                 from ../GPU/Null/NullGpu.cpp:18:
../Common/Hashmaps.h:11:7: error: 'uint32_t' does not name a type; did you mean 'u_int32_t'?
   11 | const uint32_t hashmapSeed = 0x23B58532;
      |       ^~~~~~~~
      |       u_int32_t
```

This happens due to improved transitive include hygiene on
gcc-10 headers. The change includes <cstdint> to pull in
'uint32_t' typedef.

Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>
---
 Common/Hashmaps.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Common/Hashmaps.h b/Common/Hashmaps.h
index f0a6fe4709..e5e36be02e 100644
--- a/Common/Hashmaps.h
+++ b/Common/Hashmaps.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <cstdint> /* uint32_t */
 #include <cstring>
 #include <vector>
 
